src/openforms/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from django.contrib import admin
+from django.contrib.admin import AdminSite
+from django.test import SimpleTestCase
+
+import requests
+
+from openforms.accounts.models import User
+from openforms.utils.decorators import suppress_requests_errors
+
+
+@suppress_requests_errors(fields=["first_name"])
+class TestAdmin(admin.ModelAdmin):
+    model = User
+
+    def formfield_for_dbfield(self, db_field, request, **kwargs):
+        if db_field.name in ["first_name", "last_name"]:
+            raise requests.RequestException
+
+        return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+
+class DbFieldsExceptionHandlerTests(SimpleTestCase):
+    def test_none_crashing_field(self):
+        model_admin = TestAdmin(User, AdminSite(name="test"))
+
+        formfield = model_admin.formfield_for_dbfield(
+            User._meta.get_field("username"), ...
```

Comment:
please use the django test client request factory to pass a dummy request: https://docs.djangoproject.com/en/4.2/topics/testing/advanced/#the-request-factory